src/main/java/heronarts/lx/midi/surface/MidiFighterTwister.java

```diff
@@ -490,9 +548,23 @@ public void controlChangeReceived(MidiControlChange cc) {
     switch (channel) {
       case CHANNEL_ROTARY_ENCODER:
         if (number >= DEVICE_KNOB && number <= DEVICE_KNOB_MAX) {
-            this.deviceListener.onKnob(number - DEVICE_KNOB, cc.getNormalized());
-            return;
+          int iKnob = number - DEVICE_KNOB;
+          if (this.deviceListener.isKnobRelative(iKnob)) {
+            if (note == KNOB_INCREMENT || note == KNOB_INCREMENT_FAST || note == KNOB_INCREMENT_VERYFAST) {
+              this.deviceListener.onKnobIncrement(iKnob);
+            } else if (note == KNOB_DECREMENT || note == KNOB_DECREMENT_FAST || note == KNOB_DECREMENT_VERYFAST) {
+              this.deviceListener.onKnobDecrement(iKnob);
+            } else {
+              // Knob sent absolute values but software is expecting relative values
```

Comment:
Ooh man that's weird... particularly since the note values for INCREMENT, INCREMENT_FAST etc. are all values which could very well be normal knob rotation values. If the MFT is in normal mode, it's gonna go funny and suddenly do increment/decrement things when moving through that 61-67 range.

Is there a way to protect against this? How does setting the MFT mode work?

Also thinking we might want to rename `note` to `value` here... bit confusing as it's not a note number.

Reply:
> The string of three ORs does look strange... I kept adding them on as I discovered flicking the knob sends a "higher velocity" relative change. I could change this to an inclusive range if that would look better?

I think that's fine, better to spell out the three explicit values, they are adjacent but not really a range so much as three magical values with similar meanings.

> Ok to leave the odd fall-through handling until we get the sysex stuff sorted out?

Yeah that's fine for now.